### PR TITLE
Add support chat to readme

### PR DESCRIPTION
**Pull request by nightpool**
also moved around dev stuff a little, and re-wrote outdated release info.

### CODE REVIEW

**Comment by nightpool** [comment]
feedback welcome!

File: README.md [hunk 1]
```diff
@@ -17,10 +20,12 @@ finding something going wrong. There's a long list of known issues
...
+Come join us in [the XKit repo chat](https://gitter.im/new-xkit/XKit) if you get stuck, or want some advice! There's normally a few people lurking in there any time of day.
+
+### Writing a New Extension
 Check out [the wiki page](https://github.com/new-xkit/XKit/wiki/Writing-a-New-Extension)!
 
-## Develop XKit
+### Develop XKit
 Review the project prerequisites and learn how to build XKit from source [over on the wiki](https://github.com/new-xkit/XKit/wiki/Build-XKit).
 
 ### Quickstart
```
Comment (0xazure) [inline README.md]: Quickstart should be a sub-heading of 'Develop XKit'.

File: README.md [hunk 2]
```diff
@@ -1,14 +1,17 @@
...
-If you want an official XKit release, those are provided by the [official
-download page](http://www.xkit.info/download). For this unofficial fork of
-XKit, look [here](https://github.com/new-xkit/XKit/releases).
+Currently we support [Firefox](https://addons.mozilla.org/en-US/firefox/addon/new-xkit/) and [Chrome](https://chrome.google.com/webstore/detail/new-xkit/inobiceghmpkaklcknpniboilbjmlald) officially. 
+We support Opera unofficially using the same file as the chrome extension, [which you can download here](https://github.com/new-xkit/XKit/releases).
+
+## Support
+First, [check the list of issues](https://github.com/new-xkit/XKit/issues), and see if there's something there that looks like the problem you're having. Otherwise, [file a new issue](https://github.com/new-xkit/XKit/issues) or come join [our support chat](https://gitter.im/new-xkit/support). We don't bite!
+
+[![Join the chat at https://gitter.im/new-xkit/support](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/new-xkit/support?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```
Comment (0xazure) [inline README.md]: Put the Gitter badge inline with the 'Support heading' maybe?

**Comment by 0xazure** [comment]
We should still keep a link to the dev Gitter around somewhere.  Maybe not as prominent as the support Gitter badge, but we need a link.

**Comment by bvtsang** [comment]
@0xazure The dev Gitter link is there, under the Contribute header.

**Comment by 0xazure** [comment]
Ah, I missed that.  Was looking for a link closer to 'Develop XKit'.  Where it is is fine.

File: README.md [hunk 3]
```diff
@@ -1,14 +1,17 @@
...
+[![Build Status](https://travis-ci.org/new-xkit/XKit.svg?branch=master)](https://travis-ci.org/new-xkit/XKit) 
 
 XKit is a small extension framework that powers tweaks for Tumblr.
 
 ## Get a release build
-If you want an official XKit release, those are provided by the [official
-download page](http://www.xkit.info/download). For this unofficial fork of
-XKit, look [here](https://github.com/new-xkit/XKit/releases).
+Currently we support [Firefox](https://addons.mozilla.org/en-US/firefox/addon/new-xkit/) and [Chrome](https://chrome.google.com/webstore/detail/new-xkit/inobiceghmpkaklcknpniboilbjmlald) officially. 
+We support Opera unofficially using the same file as the chrome extension, [which you can download here](https://github.com/new-xkit/XKit/releases).
```
Comment (0xazure) [inline README.md]: Nitpick: capitalize 'chrome'.

Comment (nightpool) [inline README.md]: nitpick: i hate capitalization.

File: README.md (hunk 2 above)
Comment (nightpool) [inline README.md]: will do.

**Comment by 0xazure** [comment]
Let's make the dev chat link a little more obvious with another inline bade:

`https://img.shields.io/badge/GITTER-DEV_CHAT_%E2%86%92-blue.svg`

Looks like:

**Comment by 0xazure** [comment]
lgtm :+1: 

@homu r+

File: README.md (hunk 3 above)
Comment (0xazure) [inline README.md]: I agree, but _consistency_~.

**Pull request merged**